README.md

```diff
@@ -17,7 +17,7 @@ For Node.js, Cloudinary provides an extension for simplifying the integration ev
 
 ## Setup ######################################################################
 
-npm install cloudinary
+npm install
```

Comment:
This should actually stay as `npm install cloudinary` as this readme file discusses how to install the project as a dependency in your own project.

running this command inside the cloudinary project will result in an error, but that the correct behavior.

Can you please fix this specifically so we can approve the PR?

Reply:
If you just want me to resolve this I can, but I'm wondering why you don't use  `npm i`.  Do you want the user to experience an error.  Why give them a pacakge.json if they can't isntall it per instructions?  Running `npm i` will isntall cloudinary without getting an error.  This is what I see when I run `npm install cloudinary`
rebeccapeltz@MacBook-Pro-2 cloudinary_npm % npm install cloudinary
npm ERR! code ENOSELF
npm ERR! Refusing to install package with name "cloudinary" under a package
npm ERR! also called "cloudinary". Did you name your project the same
npm ERR! as the dependency you're installing?
npm ERR! 
npm ERR! For more information, see:
npm ERR!     <https://docs.npmjs.com/cli/install#limitations-of-npms-install-algorithm>
see this link: https://docs.npmjs.com/cli/install#limitations-of-npms-install-algorithm

Reply:
just noticed that you moved cloudinary out of dependencies so `npm install cloudinary runs` - my upstream merge didn't pick that up for some reason